plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/newapp/NewApplicationWizard.java

```diff
@@ -43,7 +45,9 @@
  * @author Andre Dietisheim
  */
 public class NewApplicationWizard extends Wizard implements IWorkbenchWizard, IConnectionAwareWizard<Connection> {
-	
+
+	private static final String OPENSHIFT_EXPLORER_VIEW_ID = "org.jboss.tools.openshift.express.ui.explorer.expressConsoleView";
+
```

Comment:
Indeed, but I had no other use case and didn't where else to put it. In the spirit of YAGNI, I left it here :-)